src/libtiled/varianttomapconverter.cpp

```diff
@@ -220,6 +220,7 @@ SharedTileset VariantToMapConverter::toTileset(const QVariant &variant)
     const int columns = variantMap[QStringLiteral("columns")].toInt();
     const QString backgroundColor = variantMap[QStringLiteral("backgroundcolor")].toString();
     const QString objectAlignment = variantMap[QStringLiteral("objectalignment")].toString();
+    const unsigned transformations = variantMap[QStringLiteral("transformations")].toUInt();
```

Comment:
I'm not sure about storing the bitflags as-is in the JSON format. I know we use bitflags in the tile references, but it's a kind of special case to reduce file size. I'd say we should use named properties here.